packages/relay/tests/lib/sdkClient.spec.ts

```diff
@@ -157,4 +170,53 @@ describe('SdkClient', async function () {
       sinon.assert.calledOnce(convertGasPriceToTinyBarsStub);
     });
   });
+
+  describe('deleteFile', () => {
+    // states
+    const gasPrice = '0xad78ebc5ac620000';
+    const callerName = 'eth_sendRawTransaction';
+    const requestId = getRequestId();
+    const transaction = {
+      type: 1,
+      value: 0,
+      chainId: 0x12a,
+      gasPrice,
+      gasLimit: MAX_GAS_LIMIT_HEX,
+      data: '0x' + '00'.repeat(5121), // large contract
+    };
+
+    before(() => {
+      // mock captureMetrics
+      sinon.stub(sdkClient, 'captureMetrics').callsFake(() => {});
+    });
+
+    it('should execute deleteFile', async () => {
+      // prepare fileId
+      const signedTx = await signTransaction(transaction);
+      const transactionBuffer = Buffer.from(signedTx.substring(2), 'hex');
+      const ethereumTransactionData: EthereumTransactionData = EthereumTransactionData.fromBytes(transactionBuffer);
+      const fileId = await sdkClient.createFile(ethereumTransactionData.callData, client, requestId, callerName, '');
```

Comment:
Is fileId a valid entity here?
I don't see any stubbing to show that the response is properly mocked.

Reply:
At first I thought the same as I also didn't see any stub on this method. Turns out this spec class initialized a real client and hooked it to the current network in the environment. That's why the fileId is succesfully created. I also added some assertions to make sure fileId is a valid entity.

Reply:
That's odd, all these tests should be based on mocks.
Whereas the acceptance test should be hitting a real consensus node.

Maybe config got updated as some tests are still mocks in this class